R/deployApp.R

```diff
@@ -361,7 +361,7 @@ deployApp <- function(appDir = getwd(),
     )
     taskComplete(quiet, "Created application with id {.val {application$id}}")
   } else {
-    application <- taskStart(quiet, "Looking up application with id {.val {target$appId}}...")
+    taskStart(quiet, "Looking up application with id {.val {target$appId}}...")
```

Comment:
I don't see `guid` used here, but I assume it is further down.

Reply:
This was an unrelated mistake; we double-assigned `application` but the first assignment (from `taskStart`) was incorrect and immediately overwritten by the next statement. Found this while trying to trace the definition of `application`.